lib/cublas/wrappers.jl

```diff
@@ -323,17 +323,17 @@ for (fname, elty) in ((:cublasDgemv_v2,:Float64),
                        A::StridedCuMatrix{$elty},
                        X::StridedCuVector{$elty},
```

Comment:
If you want to change capitalization for consistency, also change `X` to `x` then.